.github/workflows/static-site-qa-template.yml

```diff
@@ -0,0 +1,20 @@
+name: Static Site Template
+
+on:   # yamllint disable-line rule:truthy
+  workflow_call:
+
+jobs:
+  qa:
+    name: Build Job
```

Comment:
Can we give these different names? Something with qa...

Reply:
i agree. `linkcheck`?

Reply:
I misinterpreted your comment. Updated the names. Give them a look.